src/databaseTests/src/test/kotlin/org/vaccineimpact/api/databaseTests/UserTests.kt

```diff
@@ -51,16 +54,16 @@ class UserTests : RepositoryTests<UserRepository>()
     fun `can retrieve user with any case username`()
     {
         given(this::addTestUser).check { repo ->
-            assertThat(repo.getUserByUsername("test.user")).isNotNull()
-            assertThat(repo.getUserByUsername("Test.User")).isNotNull()
+            assertThat(repo.getUserByUsername("test.user", PermissionSet()) as User).isNotNull()
+            assertThat(repo.getUserByUsername("Test.User", PermissionSet()) as User).isNotNull()
```

Comment:
I should have picked this up sooner: now that the interface returns a non-nullable object, this check will always pass. We should instead be checking that it doesn't throw an exception. Also, you shouldn't have to cast to User to check that something is not null?